src/main/java/at/hannibal2/skyhanni/config/features/garden/PersonalBestsConfig.java

```diff
@@ -0,0 +1,26 @@
+package at.hannibal2.skyhanni.config.features.garden;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+
+public class PersonalBestsConfig {
+    @Expose
+    @ConfigOption(
+        name = "Personal Best Increase FF",
+        desc = "Show in chat how much more FF you get from farming contest personal best bonus after beating the previous record."
+    )
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean contestPersonalBestIncreaseFF = true;
```

Comment:
those two names in the config now can have shorter names as their location name is already descriptivie. e.g. just "increaseFF" and "overflow"

Reply:
Okay, added configfixevent, shortened config names and optimized pbincrement